src/arrayMethodShift.js

```diff
@@ -5,7 +5,17 @@
  */
 function applyCustomShift() {
   [].__proto__.shift2 = function() {
-    // write code here
+    if (this.length === 0) {
+      return undefined;
+    }
+
+    const shiftItem = this[0];
```

Comment:
```suggestion
    const shiftedItem = this[0];
```

Reply:
if you say `shiftItem`, it sounds like a name for a function since it's a verb. But if you need to name variable, it needs to have a noun name